extensions/amp-wordpress-embed/amp-wordpress-embed.md

```diff
@@ -0,0 +1,55 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: Embeds a WordPress post.
+experimental: true
+bento: true
+---
+
+<!--
+Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-wordpress-embed
+
+## Behavior
+
+This extension creates an iframe and displays the [excerpt](https://make.wordpress.org/core/2015/10/28/new-embeds-feature-in-wordpress-4-4/) of a WordPress post.
+
+#### Example: Embedding a WordPress post
+
+```html
+<amp-wordpress-embed
+    data-url="https://make.wordpress.org/core/2015/10/28/new-embeds-feature-in-wordpress-4-4/"
+    layout="fixed-height"
+    height="240">
```

Comment:
It seems actually that WordPress embeds are normally embedded with a 600px width and 400px height. So I think a `fixed` layout would be a better example. 

```suggestion
    layout="fixed"
    height="400"
    width="600">
```

See:

* https://github.com/WordPress/wordpress-develop/blob/5def66daa039a71f02880c792c0908222dc9745d/src/wp-includes/embed.php#L1182
* https://github.com/WordPress/wordpress-develop/blob/5def66daa039a71f02880c792c0908222dc9745d/src/wp-includes/embed.php#L527-L548